initializers/aggregates.yml

```diff
@@ -1,6 +1,7 @@
-# - prefix: 10.0.0.0/16
-#   rir: RFC1918
-# - prefix: fd00:ccdd::/32
-#   rir: RFC4193 ULA
-# - prefix: 2001:db8::/32
-#   rir: RFC3849
+ # - prefix: 10.0.0.0/16
```

Comment:
What is the difference and why is it?

Reply:
Fixed whitespace mistakes.  I did not realize the tests remove the comments in the way you mentioned, so I was manually testing myself and made a mistake with adding comments back